Source/WebCore/css/parser/CSSTokenizer.cpp

```diff
@@ -140,16 +140,26 @@ unsigned CSSTokenizer::tokenCount()
     return m_tokens.size();
 }
 
-static bool isNewLine(UChar cc)
+bool isWhitespace(CSSParserTokenType type)
+{
+    return type == NonNewlineWhitespaceToken || type == NewlineToken;
+}
+
+bool isNewline(UChar cc)
```

Comment:
What I said about overloading and `isWhitespace` doesn't apply to this function, which takes a `UChar`. Nothing in the name of this function or the types of its arguments makes it clear that it's the CSS tokenizer definition of newline, not a globally correct version. So it would be better to leave this one as a static member function or give it a slightly longer name.

Reply:
Moved it back into `CSSTokenizer` and made it a static function.

Reply:
Also, as per Matthieu's comment below, I also made `isWhitespace` back into a static member function of `CSSTokenizer` to prevent putting a isWhitespace function that checks for tokens in the WebCore namespace, where other functions might be expecting it to check for `' '` or similar, even though it seems self explanatory that its related to the tokenizer because it takes in a `CSSParserTokenType`.

Reply:
I don’t agree with Mathieu on this, but I won’t press the point.